src/plugins/drawing/model/drawing-content.ts

```diff
@@ -192,6 +195,19 @@ export const DrawingContentModel = TileContentModel
             }
           });
         },
+        setVectorType(vectorType: VectorType, ids: string[]) {
+          self.vectorType = vectorType;
+          // FIXME not yet working
+          console.log('setting vector type to ', vectorType);
+          forEachObjectId(ids, object => {
+            if (isVectorObject(object)) {
+              console.log('setting vt on vector ', object);
+              object.setEndShapes(...endShapesForVectorType(vectorType));
+            } else {
+              console.log('Not vector object: ', object);
+            }
```

Comment:
I think this is working now, so you should remove the FIXME and `console.log`s. I also think it would be best to not log anything when an object is not a vector object in this case.